src/pages/about.astro

```diff
@@ -4,300 +4,219 @@ import CtaButtom from "../components/InlineCtaButton.astro"
+				Anishinabe Algonquin Nation. They bring an interest in quantitative
+				analysis, policy microsimulation, and competition policy to Vivic’s
+				work. Beginning in 2017 Robin has worked as a freelance consultant
+				serving several think tank and advocacy organizations. Prior to that,
+				they were the Andrew Jackson fellow at the Canadian Centre for Policy
+				Alternatives and an officer at Canada’s Competition Bureau. This year
+				Robin was a winner of the Globe and Mail’s Report on Business 2021
+				Changemakers award. They hold a MA in economics from Queen’s University
+				and is also a PhD candidate at Carleton’s School of Public Policy and
+				Administration, studying competition policy in Canada and around the
+				globe. Robin is also a research associate with the Canadian Centre
+				for Policy Alternatives.
+			</p>
+			<CtaButtom className="about-button" link="/contact">
+				Get in Touch
+			</CtaButtom>
+		</section>
+
+		<section>
+			<picture>
+				<source type="image/avif" srcset="/images/saamia.avif"/>
+				<source type="image/webp" srcset="/images/saamia.webp"/>
+				<img src="/images/saamia.jpg" alt="saamia">
+			</picture>
+			<h3>
+				Saamia Ahmad <span class="pronouns">(she/her)</span>
+			</h3>
+			<h4>
+				Economist
+			</h4>
+			<p>
+				Saamia Ahmad is an economist at Vivic Research. Saamia respectfully acknowledges
+				that she lives and works on the traditional territory of the Mississaugas
+				of the Credit, the Anishnabeg, Haudenosaunee Confederacy, and the Wendat
+				Nations. Saamia has a keen interest in public policy, with a particular
+				emphasis on how policy can be used to improve the wellbeing of all Canadians.
+				Saamia’s interests within economics are income inequality, shared prosperity,
+				and advancing human welfare through progressive economics. In her previous
+				role as a media analyst, she managed media campaigns by conducting data
+				analyses and executing programmatic strategies to streamline the overall
+				customer journey. In her role as economist at Vivic Research, Saamia develops
+				economic models and designs advanced analysis projects with the intention
+				of meaningful involvement in her projects. Saamia is a graduate of Queen’s
+				University’s Economics department.
+			</p>
+			<CtaButtom className="about-button" link="/contact">
+				Get in Touch
+			</CtaButtom>
+		</section>
+	</div>
 </MainCentered>
 
 <style>
-
-    .about-link {
-        color: rgba(246, 90, 138, 1.00);
-    }
-    
-  .big-text {
-    width: 75%;
-    font-size: 24pt;
-    font-weight: bold;
-    }
-  #about-section {
-    width: 60%;
-    display: flex;
-    flex-direction: row;
-    row-gap: 12px;
-  }
-  section {
-      padding: 0 10px;
-  }
-  section h3 {
-      font-size: 20pt;
-  }
-  section p {
-      line-height: 25px;
-  }
-
-
-  #photos {
-    grid-template-columns: auto 500px 5% 500px 5% 500px auto;
```

Comment:
The `5%` can be replaced with `gap`, and the `auto` replaced with `margin: 0 auto`. I've swapped the `500px` out for `1fr`, dropping the need for all the media queries.